src/Compilers/CSharp/Portable/Generated/CSharpSyntaxGenerator/CSharpSyntaxGenerator.SourceGenerator/Syntax.xml.Internal.Generated.cs

```diff
@@ -37109,11 +37109,19 @@ public InterpolatedStringExpressionSyntax InterpolatedStringExpression(SyntaxTok
             switch (stringStartToken.Kind)
             {
                 case SyntaxKind.InterpolatedStringStartToken:
-                case SyntaxKind.InterpolatedVerbatimStringStartToken: break;
+                case SyntaxKind.InterpolatedVerbatimStringStartToken:
```

Comment:
Nit: Is it worth pulling this out into a helper method such as `ThrowIfNotStringToken(bool isOpen, string tokenName)` given that the same check is effectively used in four different places.